chromebook-setup.sh

```diff
@@ -692,6 +700,7 @@ cmd_setup_fedora_rootfs()
     btrfs="${image/raw/btrfs}"
     dd if="${loopdev}p3" of="/var/tmp/$btrfs" conv=fsync status=progress
     losetup -D
+    umount ./tmpdir || true
```

Comment:
This PR looks good to me in general and makes sense. Just a drive by comment on something not from this PR, but the `losetup -D` just above this should probably never be called, instead we should just keep track of the loopback devices attached by this script and detach those specifically. Other applications could have other loopback devices set up and we should not detach those.

Reply:
You have reason, added to my TODO, thanks!